tool/tsh/common/app_azure.go

```diff
@@ -207,6 +259,19 @@ func (a *azureApp) RunCommand(cmd *exec.Cmd) error {
 	return nil
 }
 
+// usingMSAL returns true if the CLI is using Microsoft Authentication
+// Library (MSAL).
+func (a *azureApp) usingMSAL() bool {
+	ver, err := a.fetchCLIVersion()
+	if err != nil {
+		logger.WarnContext(a.cf.Context, "Unable to determine Azure CLI version", "error", err)
+		return false
```

Comment:
True might be a safer default in the future.

```suggestion
		return true
```